### ISSUE TITLE
ROS7 - Error parsing date/time

### ISSUE THREAD

**Issue by dBitech** [swoga/mikrotik-exporter#54]
```
2023-11-27T08:29:30-08:00 ERR failed to parse datetime error="parsing time \"2023-11-27 06:59:42\": month out of range" command_no=1 connection_no=1 metric_name=last_link_up module=interface request_no=201 sentence_no=12 target=hex1.lmk word="2023-11-27 06:59:42"
```

**Comment by isavitsky** [comment]
Hi @dBitech,
To circumvent this error I made these changes to the code and rebuilt the image:
```
diff --git a/config/param_logic.go b/config/param_logic.go
index 8f991ae..97802ec 100644
--- a/config/param_logic.go
+++ b/config/param_logic.go
@@ -9,8 +9,8 @@ import (
 )
 
 const (
-       DATETIME_FORMAT_V6 = "Jan/02/2006 15:04:05"
-       DATETIME_FORMAT_V7 = "2006-02-01 15:04:05"
+       DATETIME_LAYOUT_V6 = time.Layout
+       DATETIME_LAYOUT_V7 = time.DateTime
 )
 
 func (param *Param) PreprocessValue(log zerolog.Logger, response map[string]string, variables map[string]string) (string, bool) {
@@ -115,15 +115,13 @@ func (param *Param) tryGetValue(log zerolog.Logger, response map[string]string,
 
                return value, true
        case PARAM_TYPE_DATETIME:
+               dateTimeLayout := DATETIME_LAYOUT_V6
                startsWithNumber := len(word) > 0 && word[0] >= '0' && word[0] <= '9'
-               var dateTimeFormat string
                if startsWithNumber {
-                       dateTimeFormat = DATETIME_FORMAT_V7
-               } else {
-                       dateTimeFormat = DATETIME_FORMAT_V6
+                       dateTimeLayout = DATETIME_LAYOUT_V7
                }
 
-               dateTime, err := time.ParseInLocation(dateTimeFormat, word, time.Local)
+               dateTime, err := time.ParseInLocation(dateTimeLayout, word, time.Local)
                if err != nil {
                        parseLog.Err(err).Msg("failed to parse datetime")
                        return 0, false

**Comment by swoga** [comment]
Hi,

the fix was not released yet, I have published a new version [v0.7.0](https://github.com/swoga/mikrotik-exporter/releases/tag/v0.7.0) including the fix.
Please remember the note in th [README](https://github.com/swoga/mikrotik-exporter#info---routeros-v7).
